cmd/bosun/sched/sched.go

```diff
@@ -704,6 +725,111 @@ func (a ActionType) MarshalJSON() ([]byte, error) {
 	return json.Marshal(a.String())
 }
 
+type Incident struct {
+	Id       uint32
+	Start    time.Time
+	End      time.Time
+	AlertKey expr.AlertKey
+}
+
+func (s *Schedule) createIncident(ak expr.AlertKey, start time.Time) *Incident {
+	s.incidentLock.Lock()
+	defer s.incidentLock.Unlock()
+	if s.Incidents == nil {
```

Comment:
Because it is a field on the scedule, and we do `s := &Schedule{}` friggin everywhere.